UnitTests/CommonTestUtils/AsyncTestHelper.cs

```diff
@@ -15,64 +12,5 @@ public static TimeSpan UnexpectedTimeout
                 return Debugger.IsAttached ? TimeSpan.FromHours(1) : TimeSpan.FromMinutes(1);
             }
         }
```

Comment:
I'd move this single remaining element to `GitExtUtils/GitUI/ThreadHelper.cs` where `JoinPendingOperationsAsync(TimeSpan timeout)` was added.

Reply:
I ended up moving the other way, since `ThreadHelper` is production code.